generator/test/error_test_data/missing_underscore.expected

```diff
@@ -1,12 +1,15 @@
 Not a realm type
 
-in: asset:pkg/test/error_test_data/missing_underscore.dart:7:8
-  ╷
-5 │ @RealmModel()
-6 │ class _Bad {
-  │       ━━━━ in realm model for 'Bad'
-7 │   late Other other;
-  │        ^^^^^ Other is not a realm model type
-  ╵
-Did you intend to use _Other as type for 'other'?
+in: asset:pkg/test/error_test_data/missing_underscore.dart:8:8
+    ╷
+4   │ class Other {} // dummy stand-in for the generated class
+    │       ━━━━━ 
+... │
+6   │ @RealmModel()
+7   │ class _Bad {
+    │       ━━━━ in realm model for 'Bad'
+8   │   late Other other;
+    │        ^^^^^ Other is not a realm model type
+    ╵
+Add a @RealmModel annotation on 'Other', or an @Ignored annotation on 'other'.
```

Comment:
I will address this in a later PR. For now I think we can live with this error message.